docs/config.md

```diff
@@ -465,6 +465,14 @@ To customize these separately, use the `format_short_commit_id()` and
 'format_short_change_id(id)' = 'format_short_id(id).upper()'
 ```
 
+Operation ids can also be customized by the `format_short_operation_id()` alias:
```

Comment:
nit: drop the "also" to avoid misinterpretations (I assume you mean it as "in addition to commit ids and change ids", but it can be read as "as an alternative to using `format_short_id()`)